packages/block-editor/src/components/tabbed-sidebar/stories/index.story.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import TabbedSidebar from '../';
+
+const meta = {
+	title: 'BlockEditor/TabbedSidebar',
+	component: TabbedSidebar,
```

Comment:
```suggestion
	component: TabbedSidebar,
	tags: [ 'status-private' ],
```

Since this is [a private component](https://github.com/WordPress/gutenberg/blob/37a06bd005612adb558b3945f81e989114f90950/packages/block-editor/src/private-apis.js#L79), it would be a good idea to add a badge. See https://github.com/WordPress/gutenberg/issues/67165#issuecomment-2563701317